source/comparisons.rst

```diff
@@ -10,7 +10,28 @@ Details and porting strategies follow.
 Unorderable Types
 ~~~~~~~~~~~~~~~~~
 
-XXX
+Strict approach to comparing in Python 3 makes it impossible to compare the
+different types of objects.
+
+While this example where we try to compare ``int`` and ``str`` in Python 2
+works (with bad result)::
```

Comment:
I wouldn't call it 'bad result' – sometimes it's exactly what the user wants. I'd use 'an unpredictable result'.